spec/helpers/abbreviated_treatment_report_helper_spec.rb

```diff
@@ -13,4 +13,73 @@
 #   end
 # end
 RSpec.describe AbbreviatedTreatmentReportHelper, type: :helper do
+  let(:conservation_record) { create(:conservation_record) }
+  let(:repair_type) { create(:controlled_vocabulary, vocabulary: 'repair_type', key: 'Repair/restore binding') }
+  let(:user) { create(:user, display_name: 'Haritha Vytla') }
+  let(:repair_record) do
+    create(:in_house_repair_record,
+           performed_by_user_id: user.id,
+           repair_type: repair_type.id,
+           conservation_record: conservation_record,
+           minutes_spent: 2)
+  end
+
+  describe 'generate_abbreviated_treatment_report_type' do
+    it 'returns "Repair Type"' do
+      return_value = helper.generate_abbreviated_treatment_report_type(repair_record, 1)
+      expect(return_value).to eq('Repair/restore binding')
+    end
+  end
+
+  describe 'generate_abbreviated_treatment_report_performed_by' do
+    it 'returns the "User ID"' do
+      return_value = helper.generate_abbreviated_treatment_report_performed_by(repair_record, 1)
+      expect(return_value).to eq('Haritha Vytla')
+    end
+  end
+
+  describe 'generate_abbreviated_treatment_report_time' do
+    it 'returns "minutes spent on In House Repair Record"' do
+      return_value = helper.generate_abbreviated_treatment_report_time(repair_record, 1)
+      expect(return_value).to eq('2')
+    end
+  end
+
+  describe 'conservators_note' do
```

Comment:
I think we can remove the test for `conservators_note` and remove the entire method for `conservators_note` in app/helpers/abbreviated_treatment_report_helper.rb.  I think that method was a mistake.  The `conservators_note` method is never used.